src/plots/cartesian/axes.js

```diff
@@ -839,7 +839,7 @@ function autoTickRound(ax) {
         // 2 digits past largest digit of dtick
         ax._tickround = 2 - Math.floor(Math.log(dtick) / Math.LN10 + 0.01);
 
-        var maxend = Math.max(Math.abs(rng[0]), Math.abs(rng[1]));
+        var maxend = Math.max(Math.abs(rng[0]), Math.abs(rng[1])) || 1;
```

Comment:
Hmm. I'm not a fan of this. We shouldn't have to patch this deep in `axes.js`. If cartesian axes don't need this `|| 1` fallback, `parcoords` shouldn't need it.

Could you help understand why we need to `log` a range value of 0?

Reply:
I could simply remove this from here. But I thought we should check for that to avoid log(0).

Reply:
Ha I see, https://github.com/plotly/plotly.js/pull/4082/commits/fde0d320b747abbb3f92da79bd0fc7bbd1624509 is enough to fix the problem. Sorry for the confusion.

Reply:
Yes. That should be enough. Do you want me to undo https://github.com/plotly/plotly.js/pull/4082/commits/765ca5a62b88eb94bf64dfe32f1eb364d5aff0cb?

Reply:
If `maxend` is 0 here, it leads to a pretty serious malfunction for this routine. Just mocking it to `1` isn't really the behaviour we want. In fact, the fallback is just there to not make `Math.log` blowup. Cartesian (and polar and maybe other axes) ranges are sanatised during [`ax.cleanRange`](https://github.com/plotly/plotly.js/blob/38cd9e26e4e05c3d175c5b251b2c9a213f03ee56/src/plots/cartesian/set_convert.js#L387-L449), so `maxend=0` never happens and the fallback is never used, so I'd be ok with dropping the fallback.

That said, since we're starting to reuse the more of the `Axes` logic, I think it would be best to log and error using `Lib.error` - similar to

https://github.com/plotly/plotly.js/blob/38cd9e26e4e05c3d175c5b251b2c9a213f03ee56/src/plots/cartesian/axes.js#L935

So @archmoj you could _just_ drop it, or drop it and add a `Lib.error` when `maxend === 0`

Reply:
@etpinard thanks for your review.
Dropped condition check of `Axes` in https://github.com/plotly/plotly.js/pull/4082/commits/096ed736d27392563f47741fc482a52937bfa118.